### PR TITLE
[CU-86dumd161] add new flows

### PR DESCRIPTION
**Pull request by cgarcia-lightit**
# ⚡ Add New Flows - [CU-86dumd161](https://app.clickup.com/t/86dumd161) ⚡

## 💻 What type of change is this?

- [x] 💎 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Styling
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build / CI

## ⭐ Description

For more background, see ticket **CU-86dumd161[in-progress].**

### Before

image

### After

image

### To review
- [ ] I have tested this change locally in multiple screen sizes
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] If my task include an endpoint, I add the endpoint to Hopscotch/Postman Project
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by martinopp** [comment]
Task linked: [CU-86dumd161 Update admin panel tags based on referred partner question](https://app.clickup.com/t/86dumd161)

File: apps/eo_web/src/layouts/Footer.tsx [hunk 1]
```diff
@@ -134,6 +133,12 @@ const stupid_cancer = [
   },
   ...basicData,
 ];
+
+const cancer_buddy = [...basicData];
+const real_of_caring = [...basicData];
+const friend_family = [...basicData];
+const northwell_systems = [...basicData];
```
Comment (sgoycoechea-lightit) [inline apps/eo_web/src/layouts/Footer.tsx:140]: no About EO?

File: apps/eo_web/src/layouts/Footer.tsx [hunk 2]
```diff
@@ -134,6 +133,12 @@ const stupid_cancer = [
   },
   ...basicData,
 ];
+
+const cancer_buddy = [...basicData];
+const real_of_caring = [...basicData];
```
Comment (sgoycoechea-lightit) [inline apps/eo_web/src/layouts/Footer.tsx:138]: Should be realM of caring I think

File: apps/eo_web/src/screens/AccountCreation.tsx [hunk 3]
```diff
@@ -20,29 +20,33 @@ import {
...
 
 const getFlowFromReferral = (value: string) => {
-  const keys = Object.keys(ReferralOptions) as ReferralOptionsType[]
-  const key = keys.find(key => key === value)
-  return key ? ReferralOptions[key] : undefined
-}
+  const keys: ReferralOptionsType[] = Object.keys(ReferralOptions);
+  const key = keys.find((key) => key === value);
+  return key ? ReferralOptions[key] : undefined;
+};
```
Comment (sgoycoechea-lightit) [inline apps/eo_web/src/screens/AccountCreation.tsx:44]: nice!

**Review by sgoycoechea-lightit — APPROVED** [review]

**Pull request merged**